en/docs/security-reporting/reward-and-acknowledgement-program/hall-of-fame.md

```diff
@@ -30,22 +30,28 @@ security researcher community relationship.
 
 ||
 | :-----------|
+| [Abdennour Chakifi](https://twitter.com/dazaii111) |
```

Comment:
Let's add all the acknowledgement related to `2023` under *2023*, if the vulnerability is reported against choreo / asgardeo / cloud, let's add another sub heading under `2023` and have a separate table there.